### PR TITLE
CLI-11 # no more extra error when a command fails

### PR DESCRIPTION
**Pull request by jokeyrhyme**
### Fixed

-   CLI-11: remove extra "command unavailable" error when a command fails

### CODE REVIEW

File: test/cli.js [hunk 1]
```diff
@@ -2,25 +2,97 @@
...
 const npmBinTester = require('npm-bin-ava-tester');
 const test = require('ava');
 
 // local modules
 
+const lib = require('../lib/commands.js');
+
+// this module
+
+function copyExeSync (source /* : string */, destination /* : string */) {
```
Comment (mymattcarroll) [inline test/cli.js]: using `flow` here but i noticed this file does not have a `/* flow */` at the top. Is that intentional?

**Comment by mymattcarroll** [comment]
Just one question but other than that,

LGTM 👍

File: test/cli.js (hunk 1 above)
Comment (jokeyrhyme) [inline test/cli.js]: Probably dropped it when those ava FlowType things were going berserk. I'll double-check this.

Comment (jokeyrhyme) [inline test/cli.js]: Addressed in https://github.com/blinkmobile/cli/pull/60/commits/67ac9ae3336fb225342876f696ec891b0d18bab5

**Pull request merged**